File: 1894-merge-strings-alternately/solution.cs
public class Solution {
    public string MergeAlternately(string word1, string word2) {
        string ans = "";

        int A = 0, B = 0; 
        while(A < word1.Length && B < word2.Length){
            ans += word1[A];
            ans += word2[B];

            A++;
            B++;  
        }

        if(word1.Length > word2.Length){
            for(int i = word2.Length; i < word1.Length; i++){
                ans += word1[i];
            }
            return ans;
        }
        if(word2.Length > word1.Length){
            for(int i = word1.Length; i < word2.Length; i++){
                ans += word2[i];
            }
            return ans;
        }
        
        return ans;
    

    }
}

File: 1894-merge-strings-alternately/solution.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        String ans = ""; int n; boolean f;

        n = (word1.length() > word2.length())? word1.length() : word2.length();

        int i = 0;
        while(i<n){
            if(i >= word1.length()){
                while(i < word2.length()){
                    ans+=word2.charAt(i);
                    i++;
                }
                return ans;
            }if(i >= word2.length()){
                     while(i < word1.length()){
                    ans+=word1.charAt(i);
                    i++;
                }
                return ans;
            }

            ans+= word1.charAt(i);
            ans+= word2.charAt(i);
            i++;
        }
        return ans;
    }  
} 
